test/e2e/models/CodeServer.ts

```diff
@@ -56,6 +59,21 @@ export class CodeServer {
     return await this.page.isVisible(this.editorSelector)
   }
 
+  /**
+   * Checks if the editor is visible
+   */
+  async isConnected() {
+    await this.page.waitForLoadState("networkidle")
+
+    // Slices http://localhost:8080
+    // to localhost:8080
+    const hostWithoutHttp = CODE_SERVER_ADDRESS.slice(7)
```

Comment:
you could also do: `new URL("https://localhost:8080").host` so that http/https both work

```suggestion
    const hostWithoutHttp = new URL(CODE_SERVER_ADDRESS).host
```

Reply:
TIL! even better 🔥